src/librustc_session/options.rs

```diff
@@ -982,4 +982,6 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "Link native libraries in the linker invocation."),
     src_hash_algorithm: Option<SourceFileHashAlgorithm> = (None, parse_src_file_hash, [TRACKED],
         "hash algorithm of source files in debug info (`md5`, or `sha1`)"),
+    use_ctors: Option<bool> = (None, parse_opt_bool, [TRACKED],
```

Comment:
```suggestion
    use_ctors_section: Option<bool> = (None, parse_opt_bool, [TRACKED],
```
?

The name may be obvious in context of this PR, but "ctors" is a pretty overloaded term, so when you are looking through the list of compiler options or build system code adding this option it's not immediately clear what is this about.

Reply:
Same applies to the target spec.